cf-cli/Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM ruby:2.2-slim
+FROM alpine:3.3
```

Comment:
Alpine 3.4 uses ruby 2.3, instead of 2.2

I did not want to change such stuff.